src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1576,22 +1576,28 @@ fn resolution_failure(
                     };
                     // See if this was a module: `[path]` or `[std::io::nope]`
                     if let Some(module) = last_found_module {
-                        let module_name = collector.cx.tcx.item_name(module);
-                        let note = format!(
-                            "the module `{}` contains no item named `{}`",
-                            module_name, unresolved
-                        );
+                        let note = if partial_res.is_some() {
+                            let module_name = collector.cx.tcx.item_name(module);
+                            format!(
+                                "the module `{}` contains no item named `{}`",
```

Comment:
Should I change this error to something like ``there is no item named `{}` in the module `{}` `` to be consistent with the other error?

Reply:
```suggestion
                                "no item named `{}` in module`{}`",
```
seems concise and matches the other error.